test/e2e/terminal.test.ts

```diff
@@ -52,6 +52,8 @@ test.describe("Integrated Terminal", () => {
     await page.waitForLoadState("load")
     await page.keyboard.type(`echo '${testString}' > '${tmpFile}'`)
     await page.keyboard.press("Enter")
+    // It may take a second to process
+    await page.waitForTimeout(1000)
```

Comment:
There is no great way of knowing that the terminal received both the input and the enter and processed it (because it's a `<canvas>` element so we can't assert the text on screen) so having this timeout accounts for any slowness that might occur.